providers/ms365/resources/ms365.lr

```diff
@@ -4,10 +4,12 @@
 option provider = "go.mondoo.com/cnquery/v9/providers/ms365"
 option go_package = "go.mondoo.com/cnquery/v11/providers/ms365/resources"
 
+alias microsoft.organization = microsoft.tenant
+
 // Microsoft
 microsoft {
-  // List of organizations
-  organizations() []microsoft.organization
+  // Deprecated: use `microsoft.tenant` instead
+  organizations() []microsoft.tenant
```

Comment:
Should we be introducing a `tenants` field here as well to that there's a similar way to get all tenants?

Reply:
We only have one tenant, not required since users can just type `microsoft.tenant`